test/bigtest/tests/line-details-test.js

```diff
@@ -16,7 +16,35 @@ import LineDetailsPage from '../interactors/line-details-page';
 import OrderDetailsPage from '../interactors/order-details-page';
 
 describe('Line details test', function () {
-  setupApplication();
+  const ReceivingApp = () => (<div>Receiving</div>);
+  const AgreementsApp = () => (<div>Agreements</div>);
+  const InvoiceApp = () => (<div>Invoice</div>);
+
+  setupApplication({
+    modules: [
```

Comment:
didn't get why we need this

Reply:
Tests fail when redirecting to different apps  - https://jenkins-aws.indexdata.com/job/folio-org/job/ui-orders/job/PR-1133/3/#showFailuresLink

Reply:
but it was ok before, is smth changed?

Reply:
looks like it happened after `AppContextMenu` is added